plugins/VstBase/RemoteVstPlugin/CMakeLists.txt

```diff
@@ -37,8 +37,11 @@ add_executable(${EXE_NAME} WIN32
 if(IS_WIN)
     target_link_libraries(${EXE_NAME} ole32)
 else()
-    target_link_libraries(${EXE_NAME} pthread dl X11)
-ENDIF()
+    target_link_libraries(${EXE_NAME} dl X11)
+endif()
+if(NOT WIN32)
+	target_link_libraries(${EXE_NAME} pthread)
```

Comment:
spaces or tabs? The eternal conflict. 😁

Reply:
The majority of the file uses tabs, but there are a couple of lines that use spaces. I left the existing indentation alone, and used tabs for my additional code. I can fix it up so it's all tabs if you'd like.

Another thing to discuss is casing - all the commands visible in the diff are lowercase, but there are also a lot of uppercase commands in our CMake code. I prefer lowercase as I think it's easier to read, and it's also the recommended style now.

Reply:
Yeah I haven't touched anything in CMake styling yet. I think tabs are probably the better way to go here since they're indentations rather than alignment.
As for casing, CMake casing is all over the place. Just follow their recommendations I think.